pkg/manifest/anaconda_installer_test.go

```diff
@@ -131,3 +131,55 @@ func TestAnacondaInstallerModules(t *testing.T) {
 		})
 	}
 }
+
+func TestISOLocale(t *testing.T) {
+	pkgs := []rpmmd.PackageSpec{
+		{
+			Name:     "kernel",
+			Checksum: "sha256:eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee",
+		},
+	}
+
+	locales := []string{"C.UTF-8", "en_US.UTF-8"}
+	for _, value := range locales {
```

Comment:
Nitpick: you could have probably used `map[string]string` to capture the configured locale value as the **key** and the expected value in the stage as the **value**, and iterate over them in a single loop. Thus, effectively getting rid of the duplicate code block after the current for loop.

Reply:
You're right, I was being sloppy and lazy.  Thanks :)